src/components/pages/Home/Offerers/__specs__/OffererDetails.spec.jsx

```diff
@@ -525,4 +544,92 @@ describe('offererDetails', () => {
       ).toBeInTheDocument()
     })
   })
+
+  describe('when offerer is not yet validated', () => {
+    beforeEach(() => {
+      virtualVenue = { ...virtualVenue, nOffers: 0 }
+      const nonValidatedOfferer = {
+        ...baseOfferers[0],
+        isValidated: false,
+        managedVenues: [virtualVenue],
+      }
+      pcapi.getOfferer.mockResolvedValue(nonValidatedOfferer)
+      pcapi.getAllOfferersNames.mockResolvedValue([
+        { name: nonValidatedOfferer.name, id: nonValidatedOfferer.id },
+      ])
+    })
+
+    it('should warn user offerer is being validated', async () => {
+      // When
+      await renderHomePage()
+
+      // Then
+      expect(screen.getByText('Votre structure est en cours de rattachement')).toBeInTheDocument()
+    })
+
+    it('should allow user to view offerer informations', async () => {
+      // When
+      await renderHomePage()
+
+      // Then
+      expect(screen.getByText('Informations pratiques')).toBeInTheDocument()
+      expect(screen.getByText('Coordonnées bancaires')).toBeInTheDocument()
+    })
+
+    it('should allow user to add venue and virtual offer', async () => {
+      // When
+      await renderHomePage()
+
+      // Then
+      expect(screen.getByRole('link', { name: 'Créer un lieu' })).toBeInTheDocument()
+      expect(screen.getByRole('link', { name: 'Créer une offre numérique' })).toBeInTheDocument()
+    })
+  })
+
+  describe('when user offerer is not yet validated', () => {
```

Comment:
```suggestion
  describe('when user attachment to offerer is not yet validated', () => {
```

Ou alors tu attaches le `UserOfferer`

Mais je me dis que ça pourrait être intéressant de commencer à avoir un langage plus métier sur ce rattachement, plutôt que dire à chaque fois `UserOfferer`